test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,49 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+
+const userUrl = 'https://api.github.com/user';
+const token = process.env.ACCESS_TOKEN;
+
+const miWebIssuesUrl = 'https://api.github.com/repos/Maosorio92/miweb/issues';
+const axiosClient = axios.create({
+  headers: {
+    Authorization: `token ${token}`
+  }
+});
+describe('Methods PATCH & POST', () => {
+  let reposResponse;
+  let issueCreationResponse;
+  before(async () => {
+    const userResponse = await axiosClient.get(`${userUrl}`, {
+      headers: {
+        Authorization: `token ${token}`
+      }
+    });
+    reposResponse = await axiosClient.get(`${userResponse.data.repos_url}`);
+  });
+  it('Get Loged User', async () => {
+    expect(reposResponse.status).to.equal(StatusCodes.OK);
+    expect(reposResponse.data.find((x) => x.visibility === 'public')).to.not.equal(undefined);
+  });
+  it('Repo exists', async () => {
+    expect(reposResponse.status).to.equal(StatusCodes.OK);
+    expect(reposResponse.data.find((x) => x.name === 'miweb')).to.not.equal(undefined);
+  });
+  it('Creating Issue', async () => {
+    issueCreationResponse = await axiosClient.post(`${miWebIssuesUrl}`, {
+      title: 'Issue8'
+    });
+    expect(issueCreationResponse.data.body).to.equal(null);
+    expect(issueCreationResponse.status).to.equal(201);
+    expect(issueCreationResponse.data.title).to.equal('Issue8');
+  });
+  it('Changing Issue', async () => {
+    const issueUpdated = await axiosClient.patch(`${miWebIssuesUrl}/${issueCreationResponse.data.number}`, {
+      body: 'Learning and learningx2'
+    });
+    expect(issueUpdated.data.body).to.equal('Learning and learningx2');
+    expect(issueUpdated.status).to.equal(StatusCodes.OK);
+    expect(issueUpdated.data.title).to.equal('Issue8');
```

Comment:
add one expect to verify the body is not empty or equal to the body passed in the patch method

Reply:
=( I don´t know why is this file here...
Maybe I uploaded again by mistake, maybe when I runned the command npm run lint -- --fix, something changed and I uploaded it by mistake
you should ignore it because this was the excercise before Gist.test
Anyway, I added one expect verifying that body is not empty
=)

Reply:
oo I see, well what you need to do is, merge main into this branch and solve the conflicts locally:

- git checkout main
- git pull 
- git checkout deletemethod
- git merge main
- fix the conflicts
- add, commit and push the fix to the conflict